view/frontend/web/js/recobuy.js

```diff
@@ -48,18 +48,50 @@ define(['catalogAddToCart', 'nostojs', 'jquery'], function (addToCart, nostojs,
         Recobuy.addSkuToCart(productData, element);
     };
 
-    //Product object must have fields productId and skuId productId: 123, skuId: 321
+    // Products must be and array of objects [{'productId': '123', 'skuId': '321'}, {...}]
+    Recobuy.addMultipleProductsToCart = function (products, element) {
```

Comment:
What if the product doesn't have a sku? It should be able to add product to cart without setting skuId